client/app/email-domain/task/email-domain-task.html

```diff
@@ -1,73 +1,30 @@
-                                data-ng-click="ctrlEmailsTabTasksCtrl.refreshTasks()">
-                            <span class="fa fa-refresh" aria-hidden="true"></span>
-                        </button>
-                    </th>
-                </tr>
-            </thead>
 
-            <!-- tasks -->
-            <tbody>
-                <tr data-ng-repeat="task in (ctrlEmailsTabTasksCtrl.tasks | orderBy:['date', 'timestamp']:true) track by $index">
-                    <td data-ng-bind="task.action"></td>
-                    <td data-ng-bind="(task.account || task.name)"></td>
-                    <td data-ng-bind="task.domain"></td>
-                    <td data-ng-bind="(task.date || task.timestamp) | date:'medium'"></td>
-                    <td class="text-center"
-                        data-ng-if="task.status">
-                        <span class="label"
-                              data-ng-class="{
-                                'label-info': task.status !== 'ERROR' && task.status !== 'CANCELLED' && task.status !== 'DONE',
-                                'label-danger': task.status === 'ERROR',
-                                'label-default': task.status === 'CANCELLED',
-                                'label-success': task.status === 'DONE' }"
-                              data-ng-bind="tr('email_tab_TASK_status_' + task.status)"></span>
-                    </td>
-                    <td class="text-center"
-                        data-ng-if="!task.status">
-                        <span class="label label-info"
-                              data-ng-bind="tr('email_tab_TASK_status_DOING')"></span>
-                    </td>
-                </tr>
-            </tbody>
-        </table>
-    </div>
+    <oui-datagrid data-rows="ctrlEmailsTabTasksCtrl.taskIds"
+                  data-row-loader="ctrlEmailsTabTasksCtrl.transformItem($row)">
+        <oui-column data-title="tr('email_tab_TASK_table_function')" data-property="action"></oui-column>
+        <oui-column data-title="tr('email_tab_TASK_table_account')">
+            <span data-ng-bind="($row.account || $row.name)"></span>
+        </oui-column>
+        <oui-column data-property="domain"></oui-column>
+        <oui-column data-title="tr('email_tab_TASK_table_start_date')">
+            <span data-ng-bind="($row.date || $row.timestamp) | date:'medium'"></span>
+        </oui-column>
+        <oui-column data-title="tr('email_tab_TASK_table_state')">
+            <span class="label"
+                  data-ng-if="$row.status"
+                  data-ng-class="{
+                    'label-info': $row.status !== 'ERROR' && $row.status !== 'CANCELLED' && $row.status !== 'DONE',
+                    'label-danger': $row.status === 'ERROR',
+                    'label-default': $row.status === 'CANCELLED',
+                    'label-success': $row.status === 'DONE' }"
+                  data-ng-bind="tr('email_tab_TASK_status_' + $row.status)"></span>
```

Comment:
data-i18n-static would be a better option and single binded

Reply:
saw with u too. Canno't use this due to $row